### PR TITLE
Added app shortcut icon quality measure on Android

### PR DESCRIPTION
**Pull request by beaufortfrancois**
This PR adds a quality measure for app shortcut icons on Android.

Live preview: https://deploy-preview-3091--web-dev-staging.netlify.app/app-shortcuts/#icons-(optional)

### CODE REVIEW

File: src/site/content/en/blog/app-shortcuts/index.md [hunk 1]
```diff
@@ -136,9 +136,15 @@ include the `src` and a `sizes` property. Unlike [web app manifest icons], the
 
 SVG files are not supported at the time of writing, use PNG instead.
 
-If you want pixel-perfect icons, provide them in
-increments of 48dp. Otherwise, it is recommended that you use a 192x192 pixels
-icon.
+If you want pixel-perfect icons, provide them in increments of 48dp. Otherwise,
+it is recommended that you use a 192x192 pixels icon.
```
Comment (jpmedley) [inline src/site/content/en/blog/app-shortcuts/index.md:140]:
```suggestion
it is recommended that you use a 192x192 pixel icon.
```

File: src/site/content/en/blog/app-shortcuts/index.md [hunk 2]
```diff
@@ -136,9 +136,15 @@ include the `src` and a `sizes` property. Unlike [web app manifest icons], the
...
+
+{% Aside %}
+As a quality measure, icons must be at least half of the device's ideal size
+(48dp) on Android. This means the shortcut icon won't be visible on xxhdpi
+screens for instance if a 72x72 pixels icon is not included (`48 x (480 / 160) /
+2 = 72`).
```
Comment (jpmedley) [inline src/site/content/en/blog/app-shortcuts/index.md:146]: I don't understand this sentence.

Comment (beaufortfrancois) [reply src/site/content/en/blog/app-shortcuts/index.md:146]: Which part?

Comment (beaufortfrancois) [reply src/site/content/en/blog/app-shortcuts/index.md:146]: Shall I link to an article explaining xxhdpi displays?

Comment (jpmedley) [reply src/site/content/en/blog/app-shortcuts/index.md:146]: The icon being invisible if it's too small seems like part of the main point, but you've attached it to the example. If 48 is the device's ideal, why does 72x72 produce no result?

Comment (beaufortfrancois) [reply src/site/content/en/blog/app-shortcuts/index.md:146]: Dp and pixels are two different things.

**Comment by beaufortfrancois** [comment]
@jpmedley I've updated PR to clarify dp / pixels. How does it look now?

**Comment by rayankans** [comment]
LGTM, thanks for the clarification!

File: src/site/content/en/blog/app-shortcuts/index.md [hunk 3]
```diff
@@ -136,9 +136,14 @@ include the `src` and a `sizes` property. Unlike [web app manifest icons], the
...
 
-If you want pixel-perfect icons, provide them in
-increments of 48dp. Otherwise, it is recommended that you use a 192x192 pixels
-icon.
+If you want pixel-perfect icons, provide them in increments of 48dp (i.e. 36x36,
+48x48, 72x72, 96x96, 144x144, 192x192 pixel icons). Otherwise, it is recommended
+that you use a single 192x192 pixel icon.
+
+As a quality measure, icons must be at least half of the device's ideal size
+(48dp) on Android. This means the shortcut icon won't be visible on
```
Comment (jpmedley) [inline src/site/content/en/blog/app-shortcuts/index.md:144]:
```suggestion
As a quality measure, icons must be at least half of the device's ideal size,
which is 48dp on Android. This means the shortcut icon won't be visible on
```

File: src/site/content/en/blog/app-shortcuts/index.md [hunk 4]
```diff
@@ -136,9 +136,14 @@ include the `src` and a `sizes` property. Unlike [web app manifest icons], the
...
 
-If you want pixel-perfect icons, provide them in
-increments of 48dp. Otherwise, it is recommended that you use a 192x192 pixels
-icon.
+If you want pixel-perfect icons, provide them in increments of 48dp (i.e. 36x36,
+48x48, 72x72, 96x96, 144x144, 192x192 pixel icons). Otherwise, it is recommended
+that you use a single 192x192 pixel icon.
+
+As a quality measure, icons must be at least half of the device's ideal size,
+which is 48dp on Android. This means the shortcut icon won't be visible on
```
Comment (jpmedley) [inline src/site/content/en/blog/app-shortcuts/index.md:144]:
```suggestion
As a quality measure, icons must be at least half of the device's ideal size on
Android, which is 48dp. This means the shortcut icon won't be visible on
```

File: src/site/content/en/blog/app-shortcuts/index.md [hunk 5]
```diff
@@ -136,9 +136,14 @@ include the `src` and a `sizes` property. Unlike [web app manifest icons], the
...
-increments of 48dp. Otherwise, it is recommended that you use a 192x192 pixels
-icon.
+If you want pixel-perfect icons, provide them in increments of 48dp (i.e. 36x36,
+48x48, 72x72, 96x96, 144x144, 192x192 pixel icons). Otherwise, it is recommended
+that you use a single 192x192 pixel icon.
+
+As a quality measure, icons must be at least half of the device's ideal size on
+Android, which is 48dp. This means the shortcut icon won't be visible on
+[xxhdpi screens] for instance if a 72x72 pixel icon is not included (`48 x (480
+/ 160) / 2 = 72`).
```
Comment (jpmedley) [inline src/site/content/en/blog/app-shortcuts/index.md:146]:
```suggestion
Android, which is 48dp. For example, to display on an [xxhdpi screen], the icon must be at least 72 by 72 pixels. (This is derived from the [formula for converting] dp units for pixel units.)
```

Comment (jpmedley) [reply src/site/content/en/blog/app-shortcuts/index.md:146]: https://developer.android.com/training/multiscreen/screendensities#dips-pels

**Comment by beaufortfrancois** [comment]
@jpmedley All your feedback has been addressed.

**Comment by beaufortfrancois** [comment]
@kaycebasques Please merge if that looks good to you.

**Pull request merged**